README.md

```diff
@@ -1,3 +1,6 @@
+#2024/8/29 原始專案尚未完成 "hibernate" 功能，此更新已完成 "hibernate" 功能 (即把對應之vm當下狀態儲存至硬碟，意思就跟電腦休眠一樣，PVE主機關機後重啟也能恢復vm當下狀態繼續工作)。
```

Comment:
I believe that Chinese descriptions may not be suitable for pull requests, as the original project documentation is not in Chinese.

Reply:
Thanks for your advice.😄😄😄
The new commit is added.